Documentation/Contributors/BuildGuide/README.md

```diff
@@ -44,9 +45,26 @@ Cesium ships with a simple HTTP server for testing, run `npm start` after buildi
 
 ```
 npm start
-```
+``` 
+
+Then browse to [http://localhost:8080/](http://localhost:8080/). You should see the following:
+
+* **Apps**
+   * **Cesium Viewer** : a sample application using asynchronous module definitions (AMD) to create a Cesium 3D globe viewer with some extra features
+   * **Hello World** : an example using the combined and minified Cesium library to create a 3D globe. [Tutorial here](http://cesiumjs.org/tutorials/cesium-up-and-running/)
+   * **Sandcastle** : an app for viewing and creating [code examples](https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Hello%20World.html&label=Showcases), complete with a live preview
+   * **Cesium Inspector** : an example for using the Cesium Inspector widget to enable debugging features to better understand technical details of the scene
+   * **Timeline Demo** : demo of the Cesium Timeline widget
+* **Test Suites**
+* **Documentation**
 
-Then browse to [http://localhost:8080/](http://localhost:8080/).
+These apps have slightly different build processes. The basics:
+
+* `npm run build` will build Cesium Viewer, Sandcastle, Cesium Inspector and Timeline Demo from source. 
```

Comment:
Here and below, it sounds like these build steps are about building the apps, not Cesium.

Reword so it is clear that these build Cesium in two different ways, and each app uses one of those builds.